src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -825,5 +833,16 @@ public string RoleClaimType
         /// The default is <c>null</c>.
         /// </summary>
         public IEnumerable<string> ValidTypes { get; set; }
+
+        /// <summary>
+        /// Gets or sets a <see cref="bool"/> that will decide if messages are logged as error, when a policy evaluates to <c>false</c> during token validation.
+        /// Default value is <c>true</c> for backward compatibility of the behavior.
+        /// If set to <c>false</c>, exceptions are logged as Information and then thrown.
+        /// </summary>
+        /// <remarks>
+        /// When multiple polices are defined, all of them are tried until one succeeds and setting this property to false will reduce the noise in the logs. 
```

Comment:
>false [](http://example.com/codeflow?start=117&length=5)

supernit: <c>false</c>

to match pattern

Reply:
Updated.